tf-modules/distribution/main.tf

```diff
@@ -16,19 +21,21 @@ resource "aws_secretsmanager_secret_version" "thin_egress_urs_creds" {
 }
 
 module "thin_egress_app" {
-  source = "https://s3.amazonaws.com/asf.public.code/thin-egress-app/tea-terraform-build.18.zip"
-
-  auth_base_url              = var.urs_url
-  bucket_map_file            = aws_s3_bucket_object.bucket_map_yaml.key
-  bucketname_prefix          = ""
-  config_bucket              = var.system_bucket
-  domain_name                = var.distribution_url == null ? null : replace(replace(var.distribution_url, "/^https?:///", ""), "//$/", "")
-  permissions_boundary_name  = var.permissions_boundary_arn == null ? null : reverse(split("/", var.permissions_boundary_arn))[0]
-  private_vpc                = var.vpc_id
-  stack_name                 = "${var.prefix}-thin-egress-app"
-  stage_name                 = var.api_gateway_stage
-  vpc_subnet_ids             = var.subnet_ids
-  urs_auth_creds_secret_name = aws_secretsmanager_secret.thin_egress_urs_creds.name
+  // source = "https://s3.amazonaws.com/asf.public.code/thin-egress-app/tea-terraform-build.18.zip"
+  source = "../../../thin-egress-app/terraform"
```

Comment:
this is for testing and needs to be removed

Reply:
It can be changed once the changes to TEA's CF template are deployed - I'm in contact with TEA's maintainer about this.

Reply:
build 25 should be compatible with the changes in this PR